OpenOversight/app/templates/partials/incident_form.html

```diff
@@ -0,0 +1,38 @@
+{% import "bootstrap/wtf.html" as wtf %}
+
+<form class="form" method="post" role="form">
+			{{ form.hidden_tag() }}
+			{{ wtf.form_errors(form, hiddens="only") }}
+			{{ wtf.form_field(form.date_field, autofocus="autofocus") }}
+			{{ wtf.form_field(form.time_field) }}
+			{{ wtf.form_field(form.report_number) }}
+			{{ wtf.form_field(form.department) }}
+			{{ wtf.form_field(form.description) }}
+			{% with subform=form.address, no_remove=True %}
+				{% include "partials/subform.html" %}
+			{% endwith %}
+			<div>
+				<legend>{{ form.license_plates.label }}</legend>
+				{% for subform in form.license_plates %}
+					{% with id="js-license-plate", number=loop.index %}
+						{% include "partials/subform.html" %}
+					{% endwith %}
+				{% endfor %}
+				{# buttons are disabled until the DOM loads and click handlers are added #}
+				<button class="btn btn-success js-add-another-button" disabled>Add another license plate</button>
```

Comment:
Small bug: If click remove when there is a single license plate, then clicking "Add another license plate" doesn't do anything

Reply:
OH YEAH, I meant to fix that. 😛